src/_ert/forward_model_runner/reporting/event.py

```diff
@@ -32,7 +28,7 @@
     Start,
 )
 from _ert.forward_model_runner.reporting.statemachine import StateMachine
-from _ert.threading import ErtThread
+from _ert.threading import ErtThread, threading
```

Comment:
Should we re-import threading from `_ert.threading` or import it directly?

Reply:
:shrug:

Reply:
OK